semantic/src/site/collections/table.overrides

```diff
@@ -191,4 +189,40 @@
       word-wrap: anywhere;
     }
   }
+
+  // For list-builder tables
+  .form-element {
+    .ui.table {
+      border-collapse: separate;
+      border-spacing: 0 10px;
+      background: none;
+      border: none;
+      margin: 0;
+    }
+
+    .ui.table td {
+      background: @surfacesWhite;
+      padding: 5px;
+    }
+
+    .ui.table tr {
+      border: none;
+    }
+
+    .ui.table:not(.unstackable) tr {
+      box-shadow: 0px 0px 0px !important;
+      padding: 0.3em 0.6em;
+      margin-bottom: 0.5em;
+      border: 1px solid @surfacesMed;
+      border-radius: 0.3em;
+    }
+
+    .ui.table tr td {
+      padding: 1px !important;
+    }
+
+    // .ui.table td p {
+    //   word-wrap: anywhere;
+    // }
```

Comment:
Can these just be deleted?

Reply:
Yup, I was wondering if we would need this but didn't have any appropriate data to test it on. Have deleted now, we can add something back in later if we need it. d8ba870f